inspire_json_merger/utils/utils.py

```diff
@@ -92,3 +92,18 @@ def scan_author_string_for_phrases(s):
         retval['titles'] = [x.strip() for x in retval['titles'] if x != '']
 
     return retval
+
+
+def load_conflicts_conf(operation):
+    curr_path = os.path.dirname(__file__)
+    conf_file = os.path.abspath(
+        os.path.join(
+            curr_path,
+            os.pardir,
+            'conflicts_conf',
+            operation + '.conf')
+    )
+    with open(conf_file) as conflicts:
+        content = conflicts.read().split('\n')
+
+    return [c.split('.') for c in content]
```

Comment:
if you want to avoid the multiple list issue of the previous config format, why not simply add this preprocessing to `filter_out`?